app/controllers/football_pitches_controller.rb

```diff
@@ -13,6 +13,8 @@ def new
 
   def show
     @football_pitch = FootballPitch.find(params[:id])
+    football_pitch_type_id = @football_pitch.football_pitch_type_id
+    @football_pitch_type = FootballPitchType.find football_pitch_type_id
```

Comment:
```suggestion
    @football_pitch_type = FootballPitchType.find @football_pitch.football_pitch_type_id
```
chị thấy tạo biên shoiw vô nghĩa em ơi

ngoài ra, sao em ko dùng `include` để load cảu foootballPickType khi load FoootballPitch luôn em 

ngoài ra, em dùng `find` thì câu lệnh `return if @football_pitch` có xảy ra ko em

dùng find_by chứ
```
@football_pitch = FootballPitch.include(:fooballPitchType).find_by id: params[:id]
```

Reply:
@vanvtt-0952  cái này em tách ra là do dòng nó dài quá c ạ, mà e chỉnh xuống dòng mãi mà không được